src/cephadm/tests/test_cephadm.py

```diff
@@ -238,3 +238,20 @@ def test_registry_login(self, get_parm, call_throws):
         with pytest.raises(Exception) as e:
             cd.command_registry_login()
         assert str(e.value) == "Failed to login to custom registry @ sample-url as sample-user with given password"
+
+    def test_get_image_info_from_inspect(self):
```

Comment:
What about to test it with different combinations of "image id" and "repo_digest" empty?